test/parallel/test-net-connect-destroy.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+const common = require('../common');
+const net = require('net');
+
+const socket = new net.Socket()
```

Comment:
Linter is flagging the missing semi-colon on this line.

Reply:
Fixed. Sorry bout that.